app/src/main/res/layout/bottomsheet_attendee_check_in.xml

```diff
@@ -157,15 +158,6 @@
                 app:srcCompat="@{ checkinAttendee.isCheckedIn ? @drawable/ic_checkout : @drawable/ic_checkin }"
                 app:tint="@android:color/white" />
 
-            <ProgressBar
```

Comment:
We  are letting the user to toggle the check in status again even if already job is pending. So here no need of progressbar. Also we are showing single/double tick in attendee layout to show sync status. So this progressbar not required.